### ISSUE TITLE
Make a nice website

### ISSUE THREAD

**Issue by phw** [phw/peek#87]
Currently there is a basic Github page at http://peek.uploadedlobster.com/, but it is just using some default template and is outdated. I would like to have some nicer website, something along the line of https://gnunn1.github.io/terminix-web/ (but probably with less information, a single pager should be enough).

**Comment by bil-elmoussaoui** [comment]
@phw i can take care of that :) 
I've done Lollypop, FeedReader and Terminix website

**Comment by darim92** [comment]
I would also like to help out :)

**Comment by phw** [comment]
@bil-elmoussaoui That would be great, I like your work on Terminix. What do you need from my side? My idea is to keep it simple. Have an animation showing Peek, a quick feature overview, installation instructions and the FAQ section from the README on a separate page.

@darim92 Cool :) I am not sure how and if it makes sense to share the work, but maybe bil-elmoussaoui can say how you can assist him.

**Comment by bil-elmoussaoui** [comment]
@phw Give me some days and just create a new repository so i can keep my work on it; shouldn't take so much of time:+1:

**Comment by bil-elmoussaoui** [comment]
@phw can you ping me once it's done, so i can start working on it before the weekend

**Comment by phw** [comment]
Just created the repository and invited you: https://github.com/phw/peek-website

**Comment by bil-elmoussaoui** [comment]
@phw Thanks :+1:

**Comment by bil-elmoussaoui** [comment]
@phw Just to let you know that i have been quite busy those last days but i started working on it. Should be ready before next week :tada:

**Comment by bil-elmoussaoui** [comment]
![screenshot from 2017-04-02 14-38-16](https://cloud.githubusercontent.com/assets/7660997/24587182/0fe79fde-17b2-11e7-9ef6-976a525ea67a.png)
Should be ready pretty soon! sorry for the delay!

**Comment by phw** [comment]
@bil-elmoussaoui No problem, I am pretty busy myself right now. Looking forward to the final result :) Thanks again

**Comment by linusg** [comment]
Any update on this?

**Comment by gort818** [comment]
@bil-elmoussaoui are you still working on this?

**Comment by ferreiradaselva** [comment]
I think you can create a github site on this repository by creating a `gh-pages` branch.

**Comment by bil-elmoussaoui** [comment]
@gort818 Yeah, I'm just so busy with finals currently. But I will push the whole code by the end of this month. Sorry for the long dealy!

**Comment by phw** [comment]
@ferreiradaselva There actually is one, but @bil-elmoussaoui is working on a much better site, and we have placed it in a separate repository: https://github.com/phw/peek-website/tree/beta

Unfortunately due to both Bilal and myself struggling for finding the time this is taking some time :)
